frontend/routes/components/dateTime/dateTime.styles.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Is this file necessary?

Reply:
You're right. I removed these generated by plop styles files.